opm/io/eclipse/rst/group.cpp

```diff
@@ -77,6 +77,8 @@ RstGroup::RstGroup(const ::Opm::UnitSystem& unit_system,
     efficiency_factor(             unit_system.to_si(M::identity,              sgrp[VI::SGroup::EfficiencyFactor])),
     inj_water_guide_rate(          sgrp_value(sgrp[VI::SGroup::waterGuideRate])),
     inj_gas_guide_rate(            sgrp_value(sgrp[VI::SGroup::gasGuideRate])),
+    gas_consumption_rate(          unit_system.to_si(M::gas_surface_rate,      sgrp[VI::SGroup::GasConsumptionRate])),
+    gas_import_rate(               unit_system.to_si(M::gas_surface_rate,      sgrp[VI::SGroup::GasImportRate])),
```

Comment:
This looks a little suspicious to me.  The consumption and import rates are [UDAs](https://github.com/OPM/opm-common/blob/5c919d2d9d6f150b6a85ea9f9a5808a44595e043/opm/input/eclipse/Schedule/Group/GConSump.hpp#L36-L37) and we typically store those in output units as [outlined in the comment above](https://github.com/OPM/opm-common/blob/cdea84725ad9bdbe3282f055dce80a96b98cfdcd/opm/io/eclipse/rst/group.cpp#L60-L62).

Reply:
> This PR does not cover the case where consumption and/or import rates are actually defined via UDQs. Was kind of hoping to take that in a follow-up PR, but will start to look into it.

It's not so much that you have to handle the UDA case right now, it's more that the rest of the code by convention stores quantities that might be UDAs in output units and expect to do unit conversion on the fly when we need the actual numeric values.  If the values are stored as SI after restart, then that creates a bit of a dichotomy between base runs and restarted runs.

Reply:
Ah, I see. I'll update.